pkg/kubectl/describe.go

```diff
@@ -173,13 +173,24 @@ func (d *NamespaceDescriber) Describe(namespace, name string, describerSettings
 	}
 	resourceQuotaList, err := d.Core().ResourceQuotas(name).List(api.ListOptions{})
 	if err != nil {
-		return "", err
+		if errors.IsNotFound(err) {
```

Comment:
Why are 404's special? Seems like I'd want a best effort describe regardless of why my request failed.